packages/foam-workspace-manager/package.json

```diff
@@ -40,10 +40,13 @@
     "typescript": "^3.9.5"
   },
   "dependencies": {
+    "@types/graphlib": "^2.1.6",
     "@types/lodash": "^4.14.157",
+    "graphlib": "^2.1.8",
```

Comment:
Is it possible to store metadata about nodes & edges with this library or should they live in a separate index? E.g. links should probably contain e.g. line and column numbers + text length to support proper highlighting in the UI.

Reply:
yes good point - it's possible to add metadata to node/edges. I am currently storing the `Bubble` object in the node, and just the text of the link in the edge (but I should probably store an object there instead, to more easily prepare for extra metadata)